spring-ai-model/src/main/java/org/springframework/ai/chat/memory/ChatMemory.java

```diff
@@ -32,6 +32,11 @@ public interface ChatMemory {
 
 	String DEFAULT_CONVERSATION_ID = "default";
 
+	/**
+	 * The key to retrieve the chat memory conversation id from the context.
+	 */
+	String CHAT_MEMORY_CONVERSATION_ID_KEY = "chat_memory_conversation_id";
```

Comment:
Consider simplifying the naming:

```suggestion
	String CONVERSATION_ID = "chat_memory_conversation_id";
```

Reply:
done